napalm_yang/parsers/xml.py

```diff
@@ -7,6 +8,15 @@
 class XMLParser(BaseParser):
 
     @classmethod
+    def init_native(cls, native, device_output=[]):
```

Comment:
Ok, I think I mislead you here with my comment. This is actually done to flatten the dict that EOS throws you back when executing commands via the eapi. We might want to avoid doing this. It's mostly here to simplify how native data is passed to the parser and then how it's consumed in the `from` field.